.github/workflows/test_pr.yaml

```
@@ -63,6 +63,7 @@ jobs:
       run: |
         python --version
         pip install -q -e .[dev]
+        pip install -q gspread pandas ipython
```

```suggestion
        pip install -q gspread ipython
```

Pandas should already be installed in `[dev]`: https://github.com/google/generative-ai-python/blob/54cfd299c227fe2159b38711edae7b2abe36db61/setup.py#L53